### ISSUE TITLE
Manual test run Teamwork 1.2.0-alpha.22

### ISSUE THREAD

**Issue by HubertNastaly** [dxos/teamwork#495]
### Instructions:
* Open a new issue and use the format of 'Manual test run' + app + version in the title.
* Use the label of 'QA' for the new issue.
* Update https://github.com/dxos/gravity/blob/main/docs/content/qa/index.md with a link to the new issue.
* If any bugs are found, open new issues if they don't already exist. Add links into the test issue. Add a comment on the issue of the format "Feature: Name of feature : BROKEN"
* Update https://github.com/dxos/gravity/blob/main/docs/content/qa/index.md with a list of broken features with links to the issues.
* When a manual test is added to automation via playwright, remove it from the template.
* Test using 2 machines with 2 different browsers (Brave / Chrome / Firefox are initial targets).
* If that's not possible, use two different profiles of Chrome / Brave on a single machine.
* Reset browser local storage between test runs.
* Each daily run should be a comment in the testing tracking issue with the form below filled out.
* Each person on the QA duty should attach at least one screen recording per week.

### Status:
- [ ] Date: 4 Nov 2020
- [ ] Teamwork version: 1.2.0-alpha.22
- [ ] KUBE: wrn://dxos/application/teamwork@alpha
- [ ] Using: 1 machine
- [ ] Browser 1: Chrome
- [ ] Browser 2: Brave
- [x] [Video recording](https://drive.google.com/drive/folders/107qcV382s2umoh-itXHy4TrTa99-UPNF?usp=sharing)

Such an error appeared suddenly:

<img width="1023" alt="Screenshot 2020-12-04 at 10 40 56" src="https://user-images.githubusercontent.com/56291339/101147878-3967fd80-361d-11eb-9ff3-a195ec9e8db4.png">

### Messenger:
- Only one camera view appeared (happened only once) - when I exited and came back it was working.

### Text editor:
- Feature: Switching between editors: BROKEN #371

### Bots:
- Feature: Inviting storage bot to party: BROKEN #485

### Parties operations:
- big delay between deleting item and change seen by another user (also bin buttons not reacting to click)
It might have been the problem with my computer performance as you can see in the end of recording.

- Feature: Change party name: BROKEN #496
- Feature: Activate / deactivate party: BROKEN #498 
- Feature: Reset storage: BROKEN #360
